lib/src/value/calculation.dart

```diff
@@ -11,6 +11,7 @@ import '../utils.dart';
 import '../value.dart';
 import '../visitor/interface/value.dart';
 import '../visitor/serialize.dart';
+import 'dart:math' as math;
```

Comment:
Imports in Dart should be ordered as `dart:` imports first, then `package:` imports, then relative imports, with a blank line between each section.

See https://dart.dev/guides/language/effective-dart/style#do-place-dart-imports-before-other-imports